### PR TITLE
[14.0][MIG] account_asset_management: Migration to v14

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa 
TT26410

ping @pedrobaeza @ernestotejeda

### CODE REVIEW

**Review by ernestotejeda — COMMENTED** [review]
Code review: :+1:

**Review by ernestotejeda — COMMENTED** [review]
Squash administrative commits for reducing commit noise

**Comment by joao-p-marques** [comment]
> Squash administrative commits for reducing commit noise

Yes, I did that, it's just that there are a lot of commits here :sweat_smile:

**Review by ernestotejeda — APPROVED** [review]
Functional review: :+1:

File: account_asset_management/security/ir.model.access.csv [hunk 1]
```diff
@@ -13,3 +13,7 @@ access_account_asset_recompute_trigger_manager,account.asset.recompute.trigger,m
 access_account_asset_group_invoice,account.asset.group,model_account_asset_group,account.group_account_invoice,1,0,0,0
 access_account_asset_group_user,account.asset.group,model_account_asset_group,account.group_account_user,1,0,0,0
 access_account_asset_group_manager,account.asset.group,model_account_asset_group,account.group_account_manager,1,1,1,1
+access_account_asset_remove_user,account.asset.remove,model_account_asset_remove,account.group_account_user,1,0,0,0
+access_account_asset_remove_manager,account.asset.remove,model_account_asset_remove,account.group_account_manager,1,1,1,1
+access_account_asset_compute_user,account.asset.compute,model_account_asset_compute,account.group_account_user,1,0,0,0
+access_account_asset_compute_manager,account.asset.compute,model_account_asset_compute,account.group_account_manager,1,1,1,1
```
Comment (pedrobaeza) [inline account_asset_management/security/ir.model.access.csv]: This is not correct, as an accounting user can do the removal process or compute assets. Put only 2 lines with 1,1,1,1 for accounting users (account managers inherit from account users).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Well converted tests! Only a comment about ACL on wizards

**Comment by joao-p-marques** [comment]
Thanks @pedrobaeza!

Changes made.

**Review by pedrobaeza — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Well, that was fast. @pedrobaeza will you block a PR that adds the reporting part to this module (a port of https://github.com/OCA/account-financial-tools/pull/980)?

**Comment by pedrobaeza** [comment]
Oh, sorry, I have it blocked with no time to review an alternative. I'm still reluctant to add new dependencies, but I don't have right now the strength to block it while I haven't complied with my part. I will migrate it tomorrow with or without the dependency.

**Comment by StefanRijnhart** [comment]
I can offer to pick up that PR (create a superseding PR that fixes conflicts) and its port to this edition if you like.

**Comment by pedrobaeza** [comment]
I'll tell you tomorrow when I check current code to be feasible to strip the dependency or not. If not, the fw-port should be direct.

**Comment by StefanRijnhart** [comment]
@pedrobaeza I don't want to be too Earth centric here but when is tomorrow on the planet where you live :sunny:

**Comment by pedrobaeza** [comment]
Nothing further from reality, Stefan. I'm the one that I'm not delivering here, but blocking, so please proceed the best way you consider, as I'm still unable to perform any action here.

**Comment by StefanRijnhart** [comment]
In that case, I propose to go with the integrated reports, including the newly introduced dependency on the helper module. Because this module is included in a repository (OCA/reporting_engine) from which modules are already likely installed on any serious configuration, I don't think introducing this new dependency during the version lifecycle is that impactful. So I'll create a superseding PR for 12.0 and its port to 14.0 soon.

**Comment by yelizariev** [comment]
Please review: https://github.com/OCA/account-financial-tools/pull/1147

**Pull request merged**